site/src/components/page-sidebar.tsx

```diff
@@ -434,6 +432,7 @@ export const Sidebar: VFC<SidebarProps> = ({
             "padding-top",
             "padding-bottom",
           ]),
+          wordBreak: "break-word",
```

Comment:
I think there's 2 minor visual issues now that words can break to new lines here:
1. The arrow icon should be more to the left when words break onto a new line to be consistent with the sections where this doesn't happen
2. The focus indicator no longer stretches to the full height of the element wrapping the text of a section

<img width="296" alt="image" src="https://user-images.githubusercontent.com/42802102/174050029-b0f74e73-94d6-4f86-91d0-f8dbd57d62a4.png">

I think this could be addressed in follow-up though to help get the fixes in this PR out quicker.

Reply:
Thanks Ben - this was actually looking pretty bad in some situations, where the line would move outside of the link, so I've fixed it in https://github.com/blockprotocol/blockprotocol/pull/378/commits/be86a7f3006724c89320133e2e4b4babcf00017f

@nathggns I've left the indicator in, but hidden, as there's an effect which is doing some layout stuff with it. Haven't looked into exactly what but this change should fix the visual appearance without affecting whatever the behavior is

Reply:
The old one animated its position between links, which seems to be why it was implemented this way. I guess we don't need that behaviour

Reply:
Will look at removing it now